doozer/doozerlib/distgit.py

```diff
@@ -1286,6 +1289,48 @@ def update_build_db(self, success_flag, task_id=None, scratch=False):
                         traceback.print_exc()
                         self.logger.error(f'Unable to extract brew task information for {task_id}')
 
+    def update_konflux_db(self, build_info, success, scratch=False):
+        if scratch:
+            return
+
+        if not self.runtime.konflux_db:
+            self.logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux DB.')
+            return
+
+        self.logger.info('Storing build info in Konflux for Brew build %s', build_info['nvr'])
+        try:
+            source_repo, commitish = build_info['source'].split('#')
+
+            build_record = KonfluxBuildRecord(
+                name=self.metadata.name,
+                group=self.runtime.group,
+                version=build_info['version'],
+                release=build_info['release'],
+                assembly=self.runtime.assembly,
+                el_target=f'el{self.metadata.branch_el_target()}',
```

Comment:
I can think of cases where this is different from what's in the nvr, like in openshift rpm that builds for both el8 and el9. I think we're better off parsing it from the nvr.

Reply:
ah I guess this is specific to an image

Reply:
But imo it's better to rely on parsing it from nvr than branch_el_target, we don't want to run in an case where we build differently and our image config isn't updated or follows differently for konflux.

Reply:
Done in 98db9d51324a513771f8f4ed06b44aca4c6d217d